jax/_src/numpy/lax_numpy.py

```diff
@@ -1111,7 +1111,9 @@ def flip(m, axis: Optional[Union[int, Tuple[int, ...]]] = None):
   _check_arraylike("flip", m)
   if axis is None:
     return lax.rev(m, list(range(len(shape(m)))))
-  return lax.rev(m, [_canonicalize_axis(axis, ndim(m))])
+  if isinstance(axis, int):
+    axis = (axis,)
```

Comment:
Instead of this isinstance check, you can use `axis = _ensure_index_tuple(axis)`. This will handle a wider range of input types.

Reply:
Cool, thanks! Changing that.